src/components/uri/Visualization.tsx

```diff
@@ -173,9 +173,7 @@ function fetchLocalizationFromUser(sdk: SDK, projectId: string): Promise<Localiz
 }
 
 function getWidthDefs(mdObjectContentProperties: IVisualizationProperties): ColumnWidthItem[] | undefined {
-    return mdObjectContentProperties && mdObjectContentProperties.widthDefs
-        ? mdObjectContentProperties.widthDefs
-        : undefined;
+    return get(mdObjectContentProperties, ["controls", "columnWidths"], undefined);
```

Comment:
`undefined` is not necessary. Lodash `get` returns by default `undefined` (https://lodash.com/docs/4.17.15#get).

Reply:
fixed